Add Aspose.PDF for .NET 24.10 release notes (#360)
Aspose/releases.aspose.com: content/en/pdf/net/release-notes/2024/aspose-pdf-for-net-24-10-release-notes.md

---
id: "aspose-pdf-for-net-24-10-release-notes"
slug: "aspose-pdf-for-net-24-10-release-notes"
linktitle: "Aspose.PDF for .NET 24.10"
title: "Aspose.PDF for .NET 24.10"
weight: 120
description: "This page contains new Aspose.PDF for .NET features, enhancement, and bug fixes in 2024, version 24.10."
type: "repository"
layout: "release"
family_listing_page_title: "Aspose.PDF for .NET 24.10"
lastmod: "2024-10-14"
sitemap:
  changefreq: "monthly"
  priority: 0.7
---

{{% alert color="primary" %}}

This page contains release notes information for Aspose.PDF for .NET 24.10.

{{% /alert %}}

## Improvements and Changes

|**Key**|**Summary**|**Category**|
| :- | :- | :- |

> |PDFNET-57711|Support singing PDF documents using ECDSA|Feature|

Reviewer: ```suggestion
|PDFNET-57711|Support signing PDF documents using ECDSA|Feature|
```

> |PDFNET-57790|Underlines to be represented as CSS when converting a PDF document to HTML|Enhancement|

Reviewer: ```suggestion
|PDFNET-57790|Underlines will be rendered as CSS-style when converting PDF to HTML|Enhancement|
```

> |PDFNET-58153|Cropping the image when it is added to the PDF page|Enhancement|

Reviewer: ```suggestion
|PDFNET-58153|Cropping an image when adding it to a PDF page|Enhancement|
```
